filter9000-boinc.cpp

```diff
@@ -289,15 +296,16 @@ int main(int argc, char **argv) {
             #endif
         }
     }
+
     #ifdef BOINC
         boinc_begin_critical_section();
     #endif
     time_t elapsed = (time(NULL) - start) + elapsed_chkpoint;
     double done = (double)total;
-    double speed = done / (double) elapsed;
-    fprintf(stderr, "\nSpeed: %.2lf/s\n", speed );
+    double speed = (done / (double) elapsed) * 65535;
```

Comment:
I'm still not entirely sold on this.

The number looks bigger, but, it's not as easily comparable to the task size.